src/briefcase/windows.py

```diff
@@ -59,6 +59,8 @@ def launcher_header(self):
         This should return a suitable relative path which will find the
         bundled python for the relevant platform
         """
+        # we need the specified python to be findable relative to a path entry
+        os.environ['PATH'] += os.pathsep + os.path.join(self.dir, 'content')
```

Comment:
It's only added to the path within the python executable for the duration of the build. It doesn't affect the rest of the pc or any other apps.

It's neccesary because setuptools/pip reads the launcher header when it builds the launcher exe's and tries to find the python.exe declared there. If it can't find it, the provided header is thrown out.

Reply:
Thanks, that makes sense to me now.